packages/react-ui-kit/src/Modal/Modal.tsx

```diff
@@ -97,23 +103,88 @@ const modalBackgroundStyle: <T>(props: OverlayBackgroundProps<T>) => ObjectInter
 
 const ModalBackground = (props: OverlayBackgroundProps) => <div css={modalBackgroundStyle(props)} {...props} />;
 
+export interface ModalActionItem {
+  bold: boolean;
+  onClick: (event: React.MouseEvent<HTMLElement>) => void;
+  title: string;
+}
+
+interface ModalActions {
+  actions?: ModalActionItem[];
+}
+
+const modalActionsWrapperStyles: () => ObjectInterpolation<undefined> = () => ({
+  borderTop: '1px solid rgba(194, 194, 194, 0.2)',
+  bottom: 0,
+  display: 'flex',
+  div: {
+    '&:hover': {
+      backgroundColor: COLOR.opaque(COLOR.GRAY_DARKEN_72, 0.04),
+    },
+    // eslint-disable-next-line sort-keys-fix/sort-keys-fix
+    '&:active': {
+      backgroundColor: COLOR.opaque(COLOR.GRAY_DARKEN_72, 0.08),
+    },
+    borderRight: '1px solid rgba(194, 194, 194, 0.2)',
+  },
+  'div:first-child': {
+    borderBottomLeftRadius: 8,
+  },
+  'div:last-child': {
+    borderBottomRightRadius: 8,
+    borderRight: 0,
+  },
+  position: 'absolute',
+  width: '100%',
+});
+
+const modalActionStyles: ({bold}: {bold: boolean}) => ObjectInterpolation<undefined> = ({bold}) => ({
+  '&:hover': {
+    color: Color(COLOR.BLUE).mix(Color(COLOR.BLACK), 0.16).toString(),
+  },
+  color: COLOR.BLUE,
+  cursor: 'pointer',
+  display: 'flex',
+  flex: 1,
+  fontWeight: bold ? 'bold' : 'normal',
+  justifyContent: 'center',
+  padding: '8px 0',
+});
+
+const ModalActions: React.FC<ModalActions> = ({actions}) => (
+  <div css={modalActionsWrapperStyles()}>
+    {actions.map(action => (
+      <div key={action.title} onClick={action.onClick} css={modalActionStyles({bold: action.bold})}>
+        {action.title}
+      </div>
+    ))}
+  </div>
+);
+
 interface ModalProps {
+  actions?: ModalActionItem[];
+  fixedWidth?: number;
   fullscreen?: boolean;
   onBackgroundClick?: () => void;
   onClose?: () => void;
+  padding?: number;
```

Comment:
Can we use style instead of props for this?

Reply:
https://github.com/wireapp/wire-web-packages/pull/3071/commits/8a1ab31912dc197cf43b791d2e5b2075d730d715